lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -324,8 +334,7 @@ class CategoryRenderer {
    */
   renderScoreGauge(category, groupDefinitions) { // eslint-disable-line no-unused-vars
     const tmpl = this.dom.cloneTemplate('#tmpl-lh-gauge', this.templateContext);
-    const wrapper = /** @type {HTMLAnchorElement} */ (this.dom.find('.lh-gauge__wrapper', tmpl));
-    wrapper.href = `#${category.id}`;
+    const wrapper = this.dom.find('.lh-gauge__wrapper', tmpl);
```

Comment:
what's the point of moving this? (this same code is also in `pwa-category-renderer`)

Reply:
It was suppressing the click event for links in the description. Only the header should be a clickable link, so the href= was moved to where the header gauges are made.

Should have removed the PWA one too.

Reply:
hows that

Reply:
that didn't work (just linked to top of page instead). Added a new 'disable' class as a quick fix. but maybe a better (but bigger) change could be done?